src/katgpucbf/xbgpu/engine.py

```diff
@@ -377,10 +377,37 @@ def _populate_sensors(self) -> None:
                 )
             )
             # Dynamic sensors
+            sensors.add(
+                aiokatcp.Sensor(
+                    str,
+                    f"{output.name}.delay",
+                    "The delay settings of the inputs for this beam: (loadmcnt "
+                    "<ADC sample count when model was loaded>, delay <in seconds>, "
+                    "phase <radians>, ...)",
+                    initial_status=aiokatcp.Sensor.Status.NOMINAL,
+                )
+            )
+            sensors.add(
+                aiokatcp.Sensor(
+                    float,
+                    f"{output.name}.quantiser-gain",
+                    "Non-complex post-summation quantiser gain applied to this beam",
+                    default=0.0,
```

Comment:
Default gain of 0 doesn't make sense.
See: https://github.com/ska-sa/katgpucbf/blob/3b6cf9039403f5479e442b1d71f244f9820d4e1f/src/katgpucbf/xbgpu/engine.py#L334